spec/std/set_spec.cr

```diff
@@ -40,6 +40,19 @@ describe "Set" do
     end
   end
 
+  describe "add?" do
+    it "returns true when object is not in the set" do
```

Comment:
Please add an expectation to make sure the value was actually added to the set.

Reply:
Done.

Reply:
I meant without relying on the method that's currently being tested. Just append a `set.includes?(1).should be_true`.

Reply:
Good point, I added it in force-pushed commit.